### ISSUE TITLE
Funciones con output arguments.

### ISSUE THREAD

**Issue by sbmirandap** [franciscogazitua/IIC2113-2024#186]
hola!, sale en los posibles descuentos las funciones con output arguments, habrá descuento si ocupamos funciones como TryParse([NotNullWhen(true)] string? s, out int result) que vendría siendo una de estas funciones? o no lo es?

**Comment by RodrigoToroIcarte** [comment]
No... el descuento es porque crees tus propias funciones con output arguments.
